doc/UsersGuide/BuildingRunningTesting/Tutorial.rst

```diff
@@ -574,13 +574,277 @@ A line of severe storms brought strong winds, flash flooding, and tornadoes to t
 
 **Weather Phenomena:** Flooding and high winds
 
-   * `Storm Prediction Center (SPC) Storm Report for 20191031 <https://www.spc.noaa.gov/climo/reports/191031_rpts.html>`__ 
+   * `Storm Prediction Center (SPC) Storm Report for 20191031 
 
 .. figure:: https://github.com/ufs-community/ufs-srweather-app/wiki/Tutorial/HalloweenStorm.jpg
    :alt: Radar animation of the Halloween Storm that swept across the Eastern United States in 2019. 
 
    *Halloween Storm 2019*
 
+Data
+-------
+
+On :srw-wiki:`Level 1 <Supported-Platforms-and-Compilers>` systems, users can find data for the Halloween Storm in the usual input model data locations (see :numref:`Section %s <DataLocations>` for a list). The data can also be downloaded from the `UFS SRW Halloween Storm Case Study <https://ufs-case-studies.readthedocs.io/en/develop/2019HaloweenStorm.html>`__.
+
+Load the workflow
+---------------------
+
+To load the workflow environment, source the lmod-setup file. Then load the workflow conda environment. From the ``ufs-srweather-app`` directory, run:
+
+.. code-block:: console
+   
+   source etc/lmod-setup.sh <platform>       # OR: source etc/lmod-setup.csh <platform> when running in a csh/tcsh shell
```

Comment:
In general, I think it's best to phase out support for csh. We are already assuming bash shell in most other places. 
```suggestion
   source etc/lmod-setup.sh <platform>
```